base_fontawesome/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Base Fontawesome",
     "summary": """Up to date Fontawesome resources.""",
-    "version": "15.0.5.15.4",
+    "version": "16.0.1.0.1",
```

Comment:
We usually tried to keep the version of font awesome on the version of the module, but it is ok for me to change it

Reply:
I'd stick to that rule

Reply:
Fixed. I didn't know that rule. Thanks for your review.

(could be merged I guess now).